WatchApp Extension/Managers/ComplicationChartManager.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  ComplicationChartManager.swift
+//  WatchApp Extension
+//
+//  Created by Michael Pangburn on 10/17/18.
+//  Copyright © 2018 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import HealthKit
+import WatchKit
+
+
+private enum ComplicationChartConstants {
+    static let textInsets = UIEdgeInsets(top: 1, left: 1, bottom: 1, right: 1)
+    static let glucoseSize = CGSize(width: 1, height: 1)
+    static let glucoseLabelAttributes: [NSAttributedString.Key: Any] = [
+        .font: UIFont(name: "HelveticaNeue", size: 5)!,
+        .foregroundColor: UIColor.chartLabel
+    ]
+}
+
+private enum GlucoseLabelPosition {
+    case high
+    case low
+}
+
+final class ComplicationChartManager {
+    var data: GlucoseChartData?
+    var lastRenderDate: Date?
+    var renderedChartImage: UIImage?
+    var visibleInterval: TimeInterval = .hours(4)
+
+    private var unit: HKUnit {
+        return data?.unit ?? .milligramsPerDeciliter
+    }
+
+    func renderChartImage(size: CGSize) -> UIImage? {
+        UIGraphicsBeginImageContextWithOptions(size, false, 0.0)
+        defer { UIGraphicsEndImageContext() }
+
+        let context = UIGraphicsGetCurrentContext()!
+        drawChart(in: context, size: size)
+        let image = context.makeImage().map(UIImage.init(cgImage:))
```

Comment:
This is unreadable... it's OK to take extra lines of code if it means future readers can follow. Use a `guard` to unwrap the CGImage.